server/controllers/request.js

```diff
@@ -0,0 +1,49 @@
+const Request = require('../models/Request');
+const asyncHandler = require("express-async-handler");
+
+exports.getRequests = asyncHandler(async (req, res ) => {
+    const userId = req.query.userId;
+    Request.find({ userId })
+    .exec(function(err, requests) {
+        if (err) {
+            res.status(500);
+            throw new Error('Problem finding document.')};
+        if (!requests) {
+            res.status(404);
+            throw new Error("No request for user exists.");
+        }
+        res.status(200).json({requests});
+    });
+});
+
+exports.createRequest = asyncHandler( (req, res ) => {
+    const { userId, sitterId, date, start, end } = req.body;
```

Comment:
Please validate the request body and returna `400` if any required filed is missing